examples/unimp_arxiv.py

```diff
@@ -0,0 +1,130 @@
+import logging
+import os
+
+import torch
+from ogb.nodeproppred import PygNodePropPredDataset
+
+import torch_geometric.transforms as T
+from torch_geometric.nn import TransformerConv
+
+logger = logging.getLogger(__file__)
+logging.basicConfig(level=logging.INFO)
+
+# Get dataset
+root = os.path.join("..", "data", "OGB")
+dataset = PygNodePropPredDataset(
+    "ogbn-arxiv",
+    root,
+    transform=T.Compose([T.ToUndirected()]),
+)
+data = dataset[0]
+y_vocab_size = 40
+
+# Form masks for labels: mask gives labels being predicted
+split_idx = dataset.get_idx_split()
+train_mask = torch.zeros(data.x.size(0))
+train_mask = train_mask.scatter_(0, split_idx["train"], 1).type(torch.bool)
+
+valid_mask = torch.zeros(data.x.size(0))
+valid_mask = valid_mask.scatter_(0, split_idx["valid"], 1).type(torch.bool)
```

Comment:
I prefer
```suggestion
valid_mask[split_idx['valid']] = True
```
since I found it to be a bit easier to read.

Reply:
yep this is nicer cheers!